LambdaEngine/Source/Input/API/InputActionSystem.cpp

```diff
@@ -0,0 +1,21 @@
+#include "Input/API/InputActionSystem.h"
+#include "Input/API/Input.h"
+
+
+namespace LambdaEngine
+{
+	InputActionSystem* InputActionSystem::s_pInputActionSystem;
```

Comment:
* Pointer should be initialized to _nullptr_
* It is done implicitly in debug mode but not guaranteed in release- mode.